index.md

```diff
@@ -46,25 +48,42 @@ title: "Catalog of IDR images formatted as OME-NGFF"
         <tr>
             <td>{{ rec["OME-NGFF version"] }}</td>
             <td>
-                <a href="http://hms-dbmi.github.io/vizarr/?source={{ rec[s3key] }}">
+                <a target="_blank" href="http://hms-dbmi.github.io/vizarr/?source={{ rec[s3key] }}">
                     <img
                         alt="IDR thumbnail for image:{{image_id}}"
                         style="margin:0"
-                        src="https://idr.openmicroscopy.org/webclient/render_thumbnail/{{image_id}}/"/>
+                        {% if rec["Thumb ID"] %}
+                        src="https://idr.openmicroscopy.org/webclient/render_thumbnail/{{ rec["Thumb ID"] }}/"
+                        {% else %}
+                        src="https://idr.openmicroscopy.org/webclient/render_thumbnail/{{image_id}}/"
```

Comment:
Is there a vizarr equivalent to the `render_thumbnail` endpoint? Not an issue with this PR or the existing blog posts but it always feel a bit convoluted to have use OMERO.webclient  URL to link to a vizarr URL.

Reply:
No. In order to render anything, Vizarr needs to load all the JS libraries etc. Vizarr is a static app, so it doesn't provide any endpoints.